src/atoms/Flex/__tests__/Flex.web.test.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import View from '../../View';
+import Space from '../../Space';
+import Flex from '../index';
+
+describe('<Flex />', () => {
+  describe('flexDirection', () => {
+    it('renders a child with flexDirection attribute and flex properties correctly', () => {
+      const { container } = render(
+        <Flex flexDirection="row">
+          <View />
```

Comment:
Make it contentful. Alone View doesn't makes sense at all

Reply:
added